src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './App.css';
+import HomePage from './components/homepage';
+
+function App() {
+  return (
+    <HomePage />
```

Comment:
- [ ] Has 2 `<Route>`s, one for the BookList component and the other for the Categories component. Kindly include routes for Books and Categories component